### PR TITLE
Align Credentials for GooglePubSub, DHS_Feed, Rapid7_InsightIDR

### PR DESCRIPTION
**Pull request by jlevypaloalto**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-7162](https://jira-hq.paloaltonetworks.local/browse/CIAC-7162)

## Description
Align Credentials for GooglePubSub, DHS_Feed, Rapid7_InsightIDR

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed_test.py [hunk 1]
```diff
@@ -167,13 +159,13 @@ def test_get_none_existing_path_with_prefix(self, dict_data, path):
         output = safe_data_get(dict_data, path, prefix='TEST')
         assert output is None
 
-    data_test_get_none_existing_path_with_prefix = [
+    data_test_get_none_existing_path_with_prefix2 = [
```
Comment (JudahSchwartz) [inline Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed_test.py:162]: why the rename?

File: Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub_test.py [hunk 2]
```diff
@@ -674,7 +674,7 @@ def test_try_pull_unique_messages__unique_first_try(self, mocker):
         ) = try_pull_unique_messages(
             client, sub_name, previous_msg_ids, last_run_time, retry_times=1
         )
-        assert debug_mock.call_count == 0
+        assert debug_mock.call_count == 1  # call in "extract_acks_and_msgs" function
```
Comment (JudahSchwartz) [inline Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub_test.py:677]: didnt we discuss asserting all calls arguments were not with the unexpected log? or that they were with the expected logs?

**Review by JudahSchwartz — APPROVED** [review]
lets make the tests better before merging. debug call called 3 times is not a good test. lets assert it was called with the expected arguments

File: Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed_test.py (hunk 1 above)
Comment (jlevypaloalto) [reply Packs/FeedDHS/Integrations/DHS_Feed/DHS_Feed_test.py:162]: ruff wasnt happy that it was overriding another variable

File: Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub_test.py (hunk 2 above)
Comment (jlevypaloalto) [reply Packs/GooglePubSub/Integrations/GooglePubSub/GooglePubSub_test.py:677]: ill do it

**Pull request merged**